### ISSUE TITLE
Just reconnect my discord

### ISSUE THREAD

**Issue by m004323** [iLinkedWasRaped/AnarchyGrabber2#1]
When i open token grabber it just reconnect my app but i don't need to give my email and pass soo i dont get token on webhook. 1 week ago it was working, reconnect and login but now just reconnect.

**Comment by iLinkedWasRaped** [comment]
So it restarts your discord after injection but doesn't ask you to log in?

**Comment by m004323** [comment]
Yes

**Comment by pillowz6666** [comment]
When I open it nothing happens and people I send it to can't open it because of some kind of error in the permissions

**Comment by shlomicohen490** [comment]
Yeah, I get the same problem, when discord is reloading, it does not force log out - it is still logged in and therefore the token won't be grabbed..
